tests/Pyxsim/pyxsim.py

```diff
@@ -0,0 +1,448 @@
+# Copyright (c) 2011, XMOS Ltd, All rights reserved
+# This software is freely distributable under a derivative of the
+# University of Illinois/NCSA Open Source License posted in
+# LICENSE.txt and at <http://github.xcore.com/>
```

Comment:
I'm unsure whether this license reference should be replaced or our license checking script should be updated to allow it.  @xross, do you know why the pyxsim code was licensed under this license?

Reply:
changed

Reply:
I don't know I'm afraid..